lib/pantry/helpers/handler_helper.rb

```diff
@@ -0,0 +1,20 @@
+module Pantry
+
+  module Helpers
+
+    class HandlerHelper
+
+      def self.create_response(session_attributes: {}, message: , end_session: true)
+        response = AlexaRubykit::Response.new
+        response.add_speech message.to_s
+        session_attributes.each do |k, v|
```

Comment:
Yes, only implemented appropriate a response for LAUNCH_REQUEST so far. INTENT_REQUEST will necessitate some ability to keep track of state via the session